proxy/src/main/java/com/wavefront/agent/QueuedAgentService.java

```diff
@@ -25,6 +25,7 @@
 import com.wavefront.api.WavefrontAPI;
 import com.wavefront.api.agent.AgentConfiguration;
 import com.wavefront.api.agent.ShellOutputDTO;
+import com.wavefront.common.NamedThreadFactory;
```

Comment:
unused import?

Reply:
I've moved `NamedThreadFactory` from `com.wavefront.agent` to `com.wavefront.common`. `NamedThreadFactory` is used in these classes but the import wasn't required earlier as they were under the same package, but is needed now.